pyproject.toml

```diff
@@ -167,7 +167,8 @@ requires = [
   markers =[
     "grad: mark a test as gradcheck test",
     "jit: mark a test as torchscript test",
-    "nn: mark a test as module test"
+    "nn: mark a test as module test",
```

Comment:
@johnnv1 as a meta task to remove old `grad` / `jit` / `nn` markers ?

Reply:
yeah, I'll open an issue for, and can make it later